src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Workspace.cs

```diff
@@ -226,6 +223,8 @@ async Task ProcessInitialEvaluationDataAsync(CancellationToken cancellationToken
 
             try
             {
+                EvaluationDataAdapter evaluationData = new(snapshot.Properties);
```

Comment:
This object allows Roslyn to call us back for any property that's in ConfigurationGeneral. In future we may also allow requesting arbitrary project properties.